plugins/common/lambda-rules-registration.cpp

```diff
@@ -0,0 +1,70 @@
+#include "lambda-rules-registration.hpp"
+
+#include <iostream>
+#include <map>
+#include <memory>
+#include <string>
+
+#include "wayfire/core.hpp"
+#include "wayfire/object.hpp"
+#include "wayfire/nonstd/observer_ptr.h"
+#include "wayfire/parser/lambda_rule_parser.hpp"
+#include "wayfire/rule/lambda_rule.hpp"
+
+namespace wf
+{
+
+nonstd::observer_ptr<lambda_rules_registrations_t> lambda_rules_registrations_t::getInstance()
+{
+    auto instance = get_core().get_data<lambda_rules_registrations_t>();
+    if (instance == nullptr)
+    {
+        get_core().store_data(std::unique_ptr<lambda_rules_registrations_t>(new lambda_rules_registrations_t()));
+
+        instance = get_core().get_data<lambda_rules_registrations_t>();
+
+        if (instance == nullptr) {
+            std::cerr << "Window lambda rules: Lazy-init of lambda registrations failed." << std::endl;
```

Comment:
Do not use `std::cerr`, but rather use our logging infrastructure (i.e `LOGE`, `LOGW`, etc).

Reply:
Done